code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -32,7 +32,7 @@
 
 	spawn_flags = SPECIES_CAN_JOIN
 	appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
-	inherent_verbs = list(/mob/living/proc/shred_limb)
+	inherent_verbs = list(/mob/living/proc/shred_limb, /mob/living/carbon/human/proc/lick_wounds)
```

Comment:
I am not in favor of sergals having healing mlems as I was under the impression their maws had a fuckload of bad bacteria in their original lore, which I would like to see in SS13 lore officially but am not forcing people to have gross maws because that could be prefbreaking.

That said, sergals also having mlems somewhat pushes it for me. Sergals should not have mlems also for the reason of _every furry species having mlems_ makes mlems not as special.

I want it to be a special thing to some of our default species.

Reply:
Would you give Xenochimera mlems? No you would not. :U 

In fact, the vulp having it is kinda iffy for me too but we'll put it to discussion.

Reply:
xenochimera would be fuckin' anti-mlems. Of all the fancy vorey verbs they shouldn't have for lore reasons, licking people to cure infections is probably top of the list.
~Now, licking people's wounds to _give_ them infections on the other hand...~

Reply:
While I see we have a disagreement on the base philosophy of mlems - I believe that they should be shared and spread across all those mlem-capable - which contrasts to your value in the unique nature of mlems, I understand your position in the matter and would be happy regardless of whats changed or left the same in this PR.